test/test_intensify/test_intensify.py

```diff
@@ -389,29 +389,40 @@ def test_eval_challenger_1(self):
             test eval_challenger() - a complete intensification run with a `always_race_against` configuration
         """
         def target(x):
+            if x['a'] == 100:
+                time.sleep(1)
             return x['a']
 
-        taf = ExecuteTAFuncDict(ta=target, stats=self.stats)
+        taf = ExecuteTAFuncDict(ta=target, stats=self.stats, run_obj="runtime")
         taf.runhistory = self.rh
 
         intensifier = Intensifier(
             tae_runner=taf, stats=self.stats,
             traj_logger=TrajLogger(output_dir=None, stats=self.stats),
             rng=np.random.RandomState(12345),
-            instances=[1, 2], run_obj_time=False,
+            instances=[1, 2], run_obj_time=True, cutoff=2,
             deterministic=False, always_race_against=self.config3, run_limit=1)
 
         self.assertEqual(intensifier.n_iters, 0)
+        self.assertEqual(intensifier.stage, IntensifierStage.RUN_FIRST_CONFIG)
 
         # intensification iteration #1
-        # run incumbent first if it was not run before
+        # run first config as incumbent if incumbent is None
         config, _ = intensifier.get_next_challenger(challengers=[self.config2, self.config1],
                                                     chooser=None)
         self.assertEqual(config, self.config2)
-        self.assertEqual(intensifier.stage, IntensifierStage.RUN_INCUMBENT)
+        self.assertEqual(intensifier.stage, IntensifierStage.RUN_FIRST_CONFIG)
         # eval config 2 (=first run)
         inc, _ = intensifier.eval_challenger(challenger=config, incumbent=None, run_history=self.rh, )
         self.assertEqual(inc, self.config2)
+        self.assertEqual(intensifier.stage, IntensifierStage.RUN_INCUMBENT)
+        self.assertEqual(self.stats.inc_changed, 1)
+
+        # regular intensification begins - run incumbent first
+        config, _ = intensifier.get_next_challenger(challengers=None,  # don't need a new list here as old one is cont'd
+                                                    chooser=None)
```

Comment:
Could you please add a check here that the config is the same as the variable `inc`?